servers/rendering/renderer_rd/environment/gi.cpp

```diff
@@ -1779,9 +1775,11 @@ void GI::SDFGI::debug_probes(RID p_framebuffer, const uint32_t p_view_count, con
 		Vector3i probe_from = cascades[cascade].position / probe_cells;
 		Vector3i ofs = gi->sdfgi_debug_probe_index - probe_from;
 		if (ofs.x < 0 || ofs.y < 0 || ofs.z < 0) {
+			RD::get_singleton()->draw_list_end();
 			return;
 		}
 		if (ofs.x > SDFGI::PROBE_DIVISOR || ofs.y > SDFGI::PROBE_DIVISOR || ofs.z > SDFGI::PROBE_DIVISOR) {
+			RD::get_singleton()->draw_list_end();
```

Comment:
@clayjohn I believe this is a bug, as the early return meant the draw list was not ended